package.json

```diff
@@ -1,9 +1,20 @@
 {
   "name": "FeedHenry-App-Forms-Phase3-App-Generator",
-  "version": "1.3.2",
-  "dependencies": {},
+  "version": "1.3.3",
+  "dependencies": {
+    "backbone": "^1.3.3",
```

Comment:
For dependencies, I would recommend keeping specific versions instead of ranges in case something changes in a bug fix PR that breaks something unexpectedly.